packages/common/src/types/models.ts

```diff
@@ -27,8 +28,8 @@ export type Role = {
 };
 
 export type Booking = {
-  thingId: string;
-  userId: string;
+  thing: { name: string; description: string };
```

Comment:
```suggestion
  thing: { id: string, name: string; description: string };
```
Id could be useful here